analyzers/tests/SonarAnalyzer.UnitTest/LiveVariableAnalysis/RoslynLiveVariableAnalysisTest.cs

```diff
@@ -161,39 +161,39 @@ public void Assigned_NotLiveIn_NotLiveOut()
         [TestMethod]
         public void LongPropagationChain_LiveIn_LiveOut()
         {
-            /*    Binary -> Jump (return) ----+
-             *    declare                     |
-             *      |                         |
-             *    Binary -> Jump (return) ---+|
-             *    use & assign               ||
-             *      |                        ||
-             *    Binary -> Jump (return) --+||
-             *    assign                    |||
-             *      |                       vvv
-             *    Simple -----------------> Exit
+            /*    Entry
+             *      |
+             *    Block 1 -------+
+             *    declare        |
+             *      |            |
+             *    Block 2 ------+|
+             *    use & assign  ||
+             *      |           ||
+             *    Block 3 -----+||
+             *    assign       |||
+             *      |          vvv
+             *    Block 4 ---> Exit
              *    use
              */
             var code = @"
 var value = 0;
 if (boolParameter)
     return;
 Method(value);
-value = 42;
+value = 1;
 if (boolParameter)
     return;
-value = 42
+value = 42;
```

Comment:
Can we somehow fail on all non-compilable examples here just to produce more clean test code?